examples/complete/outputs.tf

```diff
@@ -0,0 +1,19 @@
+output "vars" {
+  value       = module.vars.vars
+  description = "vars for the component"
+}
+
+# output "settings" {
```

Comment:
if the code block is not necessary, please remove it or have comment why you need it